tests/tests.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29503.13
+# Visual Studio Version 17
+VisualStudioVersion = 17.7.34221.43

```

Comment:
Why did we change this?

Reply:
It got changed automatically when adding new project to test.sln. Solution files are

Reply:
I think we should revert this change as the pipeline might not be able to run the tests. Hopefully new added tests would still run?

Reply:
Reverted